test/extended/marketplace/marketplace_multi.go

```diff
@@ -0,0 +1,163 @@
+package marketplace
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	g "github.com/onsi/ginkgo"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	"k8s.io/apimachinery/pkg/util/wait"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+)
+
+var _ = g.Describe("[Feature:Marketplace] Marketplace support multi format operators", func() {
+
+	defer g.GinkgoRecover()
+
+	var (
+		oc            = exutil.NewCLI("marketplace", exutil.KubeConfigPath())
+		marketplaceNs = "openshift-marketplace"
+		resourceWait  = 60 * time.Second
+
+		opsrcYamltem = exutil.FixturePath("testdata", "marketplace", "opsrc", "02-opsrc.yaml")
+	)
+
+	g.AfterEach(func() {
+		//clear the sub,csv resource
```

Comment:
nit: most conventions prescribe a space between the comment directive and the first letter of the comment

Reply:
will remove

Reply:
will update